packages/loot-core/migrations/1699454123000_reports.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE reports
+  (id TEXT PRIMARY KEY,
+   name TEXT,
+   start TEXT,
+   end TEXT, 
+   mode TEXT DEFAULT 'total',
+   groupBy TEXT DEFAULT 'Category',
+   balanceType TEXT DEFAULT 'Expense',
+   interval TEXT DEFAULT 'Monthly',
+   empty INTEGER DEFAULT 0,
+   hidden INTEGER DEFAULT 0,
+   uncat INTEGER DEFAULT 0,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what's this?

Reply:
renamed to match app code.